src/components/tl/mlx5/mcast/tl_mlx5_mcast.h

```diff
@@ -90,6 +91,7 @@ typedef struct mcast_coll_comm_init_spec {
     int                               scq_moderation;
     int                               wsize;
     int                               max_eager;
+    int                               device_mem_enabled;
```

Comment:
Imo the word "device" is misleading here, because it could refer to the nic or switch, I think the word "cuda" is more explicit

Reply:
ok let me change it to CUDA